<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionUniteVenteDialog</class>
 <widget class="QDialog" name="GestionUniteVenteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter / modifier un unité</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_nom_unite">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nom de l'unité:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="input_nom_unite">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="placeholderText">
        <string>exemple: kg, kapoka, sac </string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_facteur_conversion">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>How many base units are in this selling unit? (e.g., if base is Kg and this unit is 'Gram', factor is 0.001. If this unit is '5 Kg Bag', factor is 5)</string>
       </property>
       <property name="text">
        <string>Conversion vers unité de base:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spin_box_conversion_unite_base">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>How many base units are in this selling unit? (e.g., if base is Kg and this unit is 'Gram', factor is 0.001. If this unit is '5 Kg Bag', factor is 5)</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>0.000100000000000</double>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_prix_par_unite">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Prix unitaire</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="spin_box_prix_unitaire">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="font">
      <font>
       <family>Outfit</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>109</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_annuler">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {background-color:   rgb(217, 217, 217);  
padding: 5px 10px; border: none; border-radius: 4px;}</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_enregistrer">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {background-color:  rgb(87, 182, 255);  
padding: 5px 10px; border: none; border-radius: 4px;}</string>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
